<!DOCTYPE html><html lang="en-US"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><title>Wet lab - Ali Atiia, PhD</title><link rel="shortcut icon" href="https://aliatiia.github.io/favicon.ico" type="image/x-icon"><link rel="stylesheet" href="https://aliatiia.github.io/assets/css/just-the-docs.css"> <script async src="https://www.googletagmanager.com/gtag/js?id=UA-146080138-1"></script> <script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', "UA-146080138-1"); </script> <script type="text/javascript" src="https://aliatiia.github.io/assets/js/vendor/lunr.min.js"></script> <script type="text/javascript" src="https://aliatiia.github.io/assets/js/just-the-docs.js"></script><meta name="viewport" content="width=device-width, initial-scale=1"><title>Wet lab | Ali Atiia, PhD</title><meta name="generator" content="Jekyll v3.8.6" /><meta property="og:title" content="Wet lab" /><meta property="og:locale" content="en_US" /><meta name="description" content="Computer Science, Bioinformatics, DNA/RNA Computing, DNA Nanotechnology" /><meta property="og:description" content="Computer Science, Bioinformatics, DNA/RNA Computing, DNA Nanotechnology" /><link rel="canonical" href="https://aliatiia.github.io/research/wet-lab/" /><meta property="og:url" content="https://aliatiia.github.io/research/wet-lab/" /><meta property="og:site_name" content="Ali Atiia, PhD" /> <script type="application/ld+json"> {"headline":"Wet lab","url":"https://aliatiia.github.io/research/wet-lab/","description":"Computer Science, Bioinformatics, DNA/RNA Computing, DNA Nanotechnology","@type":"WebPage","@context":"https://schema.org"}</script><body><div class="page-wrap"><div class="side-bar"><div class="site-header"> <a href="https://aliatiia.github.io/" class="site-title lh-tight">Ali Atiia, PhD</a> <button class="menu-button fs-3 js-main-nav-trigger" data-text-toggle="Hide" type="button">Menu</button></div><div class="navigation main-nav js-main-nav"><nav role="navigation" aria-label="Main navigation"><ul class="navigation-list"><li class="navigation-list-item"><a href="https://aliatiia.github.io/" class="navigation-list-link">Home</a><li class="navigation-list-item active"><a href="https://aliatiia.github.io/research/" class="navigation-list-link">Research</a><ul class="navigation-list-child-list "><li class="navigation-list-item "><a href="https://aliatiia.github.io/research/dry-lab/" class="navigation-list-link">Dry lab</a><li class="navigation-list-item active"><a href="https://aliatiia.github.io/research/wet-lab/" class="navigation-list-link active">Wet lab</a></ul><li class="navigation-list-item"><a href="https://aliatiia.github.io/software/" class="navigation-list-link">Software</a><ul class="navigation-list-child-list "></ul><li class="navigation-list-item"><a href="https://aliatiia.github.io/resume/" class="navigation-list-link">Résumé</a><ul class="navigation-list-child-list "></ul><li class="navigation-list-item"><a href="https://aliatiia.github.io/talks/" class="navigation-list-link">Talks</a><li class="navigation-list-item"><a href="https://aliatiia.github.io/publications/" class="navigation-list-link">Publications</a></ul></nav></div><footer class="site-footer"> <a href="mailto:[lastname] [at] cs [dot] mcgill [dot] ca" text-decoration: none;> <img src="/assets/images/icons/email.png" width="20" height="30" border="" hspace="3.5"> </a> <a href="https://twitter.com/"> <img src="/assets/images/icons/twitter.png" width="20" height="25" border="" hspace="3.5"> </a> <a href="https://linkedin.com/in/aliatiia"> <img src="/assets/images/icons/linkedin.png" width="22" height="25" border=""> </a> <a href="https://github.com/aliatiia"> <img src="/assets/images/icons/github.png" width="20" height="0" border="" hspace="3.5"> </a> <br><br><br><br><p class="text-very-small text-grey-dk-000 mb-4">A <a href="https://pmarsceill.github.io/just-the-docs/">Patrick Marsceill</a> theme under <a href="https://github.com/pmarsceill/just-the-docs/tree/master/LICENSE.txt">MIT license.</a></p></footer></div><div class="main-content-wrap js-main-content" tabindex="0"><div class="main-content"><div class="page-header js-page-header"><div class="search"><div class="search-input-wrap"> <input type="text" class="js-search-input search-input" tabindex="0" placeholder="Search Ali Atiia, PhD" aria-label="Search Ali Atiia, PhD" autocomplete="off"> <svg width="14" height="14" viewBox="0 0 28 28" xmlns="http://www.w3.org/2000/svg" class="search-icon"><title>Search</title><g fill-rule="nonzero"><path d="M17.332 20.735c-5.537 0-10-4.6-10-10.247 0-5.646 4.463-10.247 10-10.247 5.536 0 10 4.601 10 10.247s-4.464 10.247-10 10.247zm0-4c3.3 0 6-2.783 6-6.247 0-3.463-2.7-6.247-6-6.247s-6 2.784-6 6.247c0 3.464 2.7 6.247 6 6.247z"/><path d="M11.672 13.791L.192 25.271 3.02 28.1 14.5 16.62z"/></g></svg></div><div class="js-search-results search-results-wrap"></div></div></div><div class="page"><nav class="breadcrumb-nav"><ol class="breadcrumb-nav-list"><li class="breadcrumb-nav-list-item"><a href="https://aliatiia.github.io/research/">Research</a><li class="breadcrumb-nav-list-item"><span>Wet lab</span></ol></nav><div id="main-content" class="page-content" role="main"><h2 id="overview"> <a href="#overview" class="anchor-heading"><svg class="d-inline-block v-align-middle" viewBox="0 0 16 16" version="1.1" width="18" height="18" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a> Overview:</h2><p>The resilient and ancient deoxyribonucleic acid (DNA) encodes the blueprint of the machinary of life. But it has in recent decades attracted interest for material purposes.</p><p>In one avenue of research, DNA has proven to be a sturdy and, more importantly, programmable substrate with which intricate 2D and 3D <a href="https://pubs.acs.org/doi/abs/10.1021/acs.chemrev.6b00825" target="_blank">nano-structures</a> can be fabricated, with exciting <a href="https://www.researchgate.net/profile/Hanadi_Sleiman/publication/320929550_DNA_nanotechnology/links/5a05b0d3aca2726b4c7c0cb1/DNA-nanotechnology.pdf" target="_blank">potential usecases</a>.</p><p>In another avenue, DNA has been used as a computational medium. Adleman kicked off this line of this research with <a href="https://www.jstor.org/stable/pdf/2885489.pdf" target="_blank">an insightful demonstration</a> of a DNA-based computation of an instance of the NP-Complete <a href="https://en.wikipedia.org/wiki/Hamiltonian_path_problem" target="_blank">Hamiltonian Path Problem (HPP)</a>, inspiring many subsequent demonstrations.</p><p>Below is a brief about my own work in both avenues.</p><hr /><h3 id="dna-knitting"> <a href="#dna-knitting" class="anchor-heading"><svg class="d-inline-block v-align-middle" viewBox="0 0 16 16" version="1.1" width="18" height="18" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a> DNA Knitting:</h3><p>“Knitting” because the basic principle is that <em>different</em> nano-structures can be “knitted” (or carved out) from the <em>same</em> template. Other aims of this method: 1) no viral scaffolding DNA involved (hence, no regulatory and lower fabrication cost), 2) maximum resolution (a pixel = one base-pair), and 3) PCR-driven (no chemical synthesis except for primers).</p><div><div class="table-wrapper"><table><tr><td width="30%"><center><img src="/assets/images/figs/nano/knitting.png" /></center><td style="text-align:justify;vertical-align:middle, font-size:1"> Programmability in DNA knitting method. Given a DNA lattice as template, segments (light area) of the horizontal (black) and vertical (red) DNA strands can be carved using PCR, effectively fabricating of nano-structures of interest ("programmability"). The choice of primer oligonucleotides (blue dots) determines the resulting nano-shape (primers ~ program arguments). Regions falling outside the range a given pair of primers are effectively “amplified-out”. &nbsp; <a href="https://github.com/aliatiia/publications/blob/master/Reports/DNA_knitting.pdf" target="_blank">Full report .. </a></table></div></div><p>These slides provide an overview of the method, for full details see <a href="https://github.com/aliatiia/publications/blob/master/Reports/DNA_knitting.pdf" target="_blank"> this report</a>. The inspiration for this method originated from a <a href="https://spectrum.library.concordia.ca/982709/" target="_blank">DNA-based computation</a> of an instance of the NP-complete edge-matching puzzles.</p><p><a href="https://github.com/aliatiia/publications/blob/master/Reports/reAdleman.pdf" target="_blank"></a></p><hr /><h3 id="dnarna-computing"> <a href="#dnarna-computing" class="anchor-heading"><svg class="d-inline-block v-align-middle" viewBox="0 0 16 16" version="1.1" width="18" height="18" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a> DNA/RNA Computing:</h3><p>Subsequent to Adleman’s <a href="https://www.jstor.org/stable/pdf/2885489.pdf" target="_blank">insightful demonstration</a> of a DNA-based method to encode and solve an instance of the NP-complete Hamiltonian Path Problem, it was realized that, while its “bit” is extremely tiny compared those in silicon-based computers, DNA will not push the limits of what can be solved practically.</p><p>The inherent intractability of NP-Complete problems still manifests in DNA computing manifests itself by requiring an exponential molar amount of DNA strands to guarantee a successful computation, analogous to how <i>in-silico</i> NPC algorithms require CPU/memory resources that are exponential in problem size. To solve a 200-node HPP instance following Adleman’s method, for example, would require an amount of raw DNA that is greater than Earth in mass (Hartmanis, Juris. “On the weight of computations.” EATCS Bulletin 55 (1995): 136-138. See also p. 70, lower paragraph, in <a href="https://mcgill.on.worldcat.org/oclc/1030147018" targert="_blank">this thesis</a> and section 1.4 in <a href="https://spectrum.library.concordia.ca/982709/" target="_blank">this thesis</a>).</p><p>Nonetheless, bringing a computational <em>perspective</em> to a soupe of interacting bio-molecules inspired many computer scientists to bring their theory into the cell (notice how this paradigm is the inverse of bioinformatics, in which the cell is “taken to the computer”).</p><div><div class="table-wrapper"><table><tr><td><center><img src="/assets/images/figs/molcomputing/cellmimicing.png" /></center><tr><td style="text-align:justify;vertical-align:middle, font-size:1"> A DNA/RNA/Protein-based HPP solution that mimics the functioning of the cell: left to right: dsDNA templates for each node &amp; edge in A-HPP, each template primed with leading (grey) sequence and T7 promoter (blue) sequence. T7 polymerase (blue circle) transcribes an RNA strand from each DNA template (orange lines for nodes; purple for edges), N<sub>0</sub>’s sequence is primed with m7G analog (<a href="https://www.neb.ca" target="_blank">NEB</a>) and contains a leading <a href="https://www.nature.com/articles/nbt.1556" target="_blank"> ribosome-binding site </a>(black vertical bars), N<sub>0</sub>’s transcript is polyadenylated (curvy dotted line) with <i>E. coli</i> poly(A) (NEB). RNA transcripts are ligated with T4 RNA ligase 2 (NEB) (orange circles) leading a brute-force exhaustive generation of correct and erroneous paths (black &amp; red subscripts, respectively). Ligated sequences beginning with N<sub>0</sub> and ending with N<sub>6</sub> represent mRNAs suitable for translation by the ribosome (by virtue of m7G cap &amp; RBS for translation initiation in N<sub>0</sub>, and poly(a) tail for mRNA stability in N<sub>6</sub>), but only the correct path encodes the EGFP protein (by deliberate design); since nodes’ sequences have different lengths, the correct solution’s protein has a unique kD weight; correctness of node sequence in the solution path is validated by the fluorescence of EGFP. &nbsp; <a href="https://github.com/aliatiia/publications/blob/master/Reports/Cell_mimicking.pdf" target="_blank">Full report .. </a></table></div></div></div></div></div></div>
